pages/content-ui/vite.config.mts

```diff
@@ -1,24 +1,38 @@
 import { resolve } from 'node:path';
-import { makeEntryPointPlugin } from '@extension/hmr';
 import { isDev, withPageConfig } from '@extension/vite-config';
+import { processCssPlugin } from '@extension/hmr';
+import { getEntryPoints } from './script/entryPoints';
 
 const rootDir = resolve(__dirname);
 const srcDir = resolve(rootDir, 'src');
 
+/**
+ * Here is the place to add new entry points, u can straight away add new folders here to include
+ * without creating folders in src dir manually.
+ * @param {string[]} options.include - Folders to include !!MODIFY ME!!
+ * @param {string[]} options.exclude - Folders to exclude !!MODIFY ME!!
+ * @param {boolean} options.createRecursively - Create folders recursively
+ * @returns {Record<string, string>}
+ */
+const entryPoints = getEntryPoints({ 
+  include: ['page1', 'page2'], createRecursively: true 
+});
+
 export default withPageConfig({
   resolve: {
     alias: {
       '@src': srcDir,
     },
   },
-  plugins: [isDev && makeEntryPointPlugin()],
+  plugins: [processCssPlugin()],
   publicDir: resolve(rootDir, 'public'),
   build: {
     lib: {
-      entry: resolve(srcDir, 'index.tsx'),
+      entry: entryPoints,
       name: 'contentUI',
-      formats: ['iife'],
-      fileName: 'index',
+      formats: ['es'],
```

Comment:
Don't have it to `es` because it isn't working good, i've checked it XD

Reply:
hmm my repo worked fine, but im really puzzled how the test all failed, i didn't even touched the content-runtime stuff

Reply:
Try to e.g. inject runtime script several times, you will have an error